cfme/common/vm.py

```diff
@@ -438,6 +438,38 @@ def unset_ownership(self):
         view = self.create_view(navigator.get_class(self, 'Details').VIEW)
         view.flash.assert_success_message(msg)
 
+    def rename_vm(self, new_vm_name, old_vm_name, click_cancel=False, click_reset=False):
+        """Rename the VM
+
+        Args:
+            new_vm_name: object for renaming vm
+            old_vm_name: object for assertion
+            click_cancel (bool): Whether to cancel form submission
+            click_reset (bool): Whether to reset form after filling
+        """
+        view = navigate_to(self, 'RenameVM', wait_for_view=0)
```

Comment:
Required:
we use `wait_for_view=0` only in such a condition when we are not in a position to check `is_displayed`. As per my understanding, you have `is_displayed` method.